scripts/ci/Jenkinsfile

```
@@ -36,7 +36,7 @@ pipeline {
                 sh './scripts/run_yamlcheck.sh'
             }
         }
-        stage ('CSI-controller & node: Build and push images') {
+        stage ('CSI-controller & node & hostDefiner: Build and push images') {
```

isn't too long?
can we
```suggestion
        stage ('Build and push images') {
```
?

---

done